src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a href="#home">
+          <img
+            src="./images/logo.png"
+            alt="Moyo logo"
+            class="logo"
+          >
+        </a>
+        <nav class="nav">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML. Fix it everywhere 
```suggestion
        </a>
        
        <nav class="nav">
```

Reply:
thanks! 
but I don't really understand how to resolve the issue with the shrinking menu and logo. I've tried to see it as you did, but for me it has been shown non-shrinked logo, so a normal one. at the same time I did see a problem with the second line appear in the menu and don't know how to fix it :( I've tried flex-wrap/flex-shrink and it still doesn't work. so far I've added flex-shrink: 0 property to the logo but can't see any difference. I've also tried to add flex-wrap: now-rap to the container with the menu -> it didn't work. then I tried to add the same property to the nav__list -> it didn't work. the same situation with flex-shrink there. so, what should I do at this point then? I'm also a bit confused with empty lines between multiline sibling block, which means I'm not sure what the multiline sibling blocks are. I added empty lines between <li> in the navigation, but not sure if you meant exactly those ones. and yes, added that empty line between logo and nav